### PR TITLE
testing/mozjs68: new aport

### PR DESCRIPTION
**Pull request by J0WI**

### CODE REVIEW

File: testing/mozjs68/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+		armv7)
+			export RUST_TARGET="armv7-unknown-linux-musleabihf"
+			;;
+		armhf)
+			export RUST_TARGET="arm-unknown-linux-musleabihf"
+			;;
+		ppc64le)
+			export RUST_TARGET="powerpc64le-unknown-linux-musl"
+			;;
+	esac
```
Comment (Cogitri) [inline testing/mozjs68/APKBUILD]: I think you're using an older version of the fix-rust-target.patch where you don't have to export RUST_TARGET

Comment (J0WI) [reply testing/mozjs68/APKBUILD]: Done. But I still messing with paths

**Comment by Cogitri** [comment]
Please request a review when you're ready

**Comment by maxice8** [comment]
Any progress ? 

@Cogitri would be nice to get this on 3.11 ?

**Comment by Cogitri** [comment]
As of now it's not necessary, but polkit might require it down the line.

**Comment by J0WI** [comment]
It's important because mozjs60 is EOL

**Comment by Cogitri** [comment]
Yes, but if applications don't switch to mozjs68's API we can't do anything about that

**Pull request closed without merging**